PodcastTests/PodcastTests.swift

```diff
@@ -20,17 +20,12 @@ class PodcastTests: XCTestCase {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
         super.tearDown()
     }
-    
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
-    }
-    
-    func testPerformanceExample() {
-        // This is an example of a performance test case.
-        self.measure {
-            // Put the code you want to measure the time of here.
-        }
+
+    func testKeys() {
```

Comment:
when will this be run?

Reply:
It is run automatically when the test suite is run